Maven4/whatsnewinmaven4.md

```diff
@@ -275,6 +276,11 @@ If a plugin still relies on long time deprecated and now removed Plexus dependen
 
 **Advice**: If you are maintaining a Maven plugin, you should test it with Maven 3.9.x, have a close look at upcoming warnings and update the plugin.
 
+## TBD
```

Comment:
Will research those things (on Wednesday) to add more than points to the article.